<html xmlns:string="xalan://java.lang.String" xmlns:lxslt="http://xml.apache.org/xslt">
    <head>
        <META http-equiv="Content-Type" content="text/html; charset=UTF-8">
        <title>Standard Error from JUnitTclTest</title>
    </head>
    <body>
        <pre>java.lang.RuntimeException: Execution failed
	at ptolemy.moml.MoMLSimpleApplication$UnloadThread.run(MoMLSimpleApplication.java:402)
Caused by: ptolemy.kernel.util.IllegalActionException: Failed to open connection to the database.
  in .DOPCSV.DatabaseManager
	at ptolemy.actor.lib.database.DatabaseManager.getConnection(DatabaseManager.java:566)
	at ptolemy.actor.lib.database.DatabaseManager.preinitialize(DatabaseManager.java:583)
	at ptolemy.actor.Director.preinitialize(Director.java:1302)
	at ptolemy.actor.Director.preinitialize(Director.java:1257)
	at ptolemy.domains.sdf.kernel.SDFDirector.preinitialize(SDFDirector.java:763)
	at ptolemy.actor.CompositeActor.preinitialize(CompositeActor.java:1828)
	at ptolemy.actor.Manager.preinitializeAndResolveTypes(Manager.java:1040)
	at ptolemy.actor.Manager.initialize(Manager.java:710)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.Manager.run(Manager.java:1263)
	at ptolemy.actor.Manager$PtolemyRunThread.run(Manager.java:1933)
java.lang.RuntimeException: Execution failed
	at ptolemy.moml.MoMLSimpleApplication$UnloadThread.run(MoMLSimpleApplication.java:402)
Caused by: ptolemy.kernel.util.IllegalActionException: Failed to open connection to the database.
  in .DOPCenter.DatabaseManager
	at ptolemy.actor.lib.database.DatabaseManager.getConnection(DatabaseManager.java:566)
	at ptolemy.actor.lib.database.DatabaseManager.preinitialize(DatabaseManager.java:583)
	at ptolemy.actor.Director.preinitialize(Director.java:1302)
	at ptolemy.actor.Director.preinitialize(Director.java:1257)
	at ptolemy.domains.sdf.kernel.SDFDirector.preinitialize(SDFDirector.java:763)
	at ptolemy.actor.CompositeActor.preinitialize(CompositeActor.java:1828)
	at ptolemy.actor.Manager.preinitializeAndResolveTypes(Manager.java:1040)
	at ptolemy.actor.Manager.initialize(Manager.java:710)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.Manager.run(Manager.java:1263)
	at ptolemy.actor.Manager$PtolemyRunThread.run(Manager.java:1933)
java.lang.RuntimeException: Execution failed
	at ptolemy.moml.MoMLSimpleApplication$UnloadThread.run(MoMLSimpleApplication.java:402)
Caused by: ptolemy.kernel.util.IllegalActionException: Failed to open connection to the database.
  in .Placard.DatabaseManager
	at ptolemy.actor.lib.database.DatabaseManager.getConnection(DatabaseManager.java:566)
	at ptolemy.actor.lib.database.DatabaseManager.preinitialize(DatabaseManager.java:583)
	at ptolemy.actor.Director.preinitialize(Director.java:1302)
	at ptolemy.actor.Director.preinitialize(Director.java:1257)
	at ptolemy.domains.sdf.kernel.SDFDirector.preinitialize(SDFDirector.java:763)
	at ptolemy.actor.CompositeActor.preinitialize(CompositeActor.java:1828)
	at ptolemy.actor.Manager.preinitializeAndResolveTypes(Manager.java:1040)
	at ptolemy.actor.Manager.initialize(Manager.java:710)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.Manager.run(Manager.java:1263)
	at ptolemy.actor.Manager$PtolemyRunThread.run(Manager.java:1933)
</pre>
    </body>
</html>
